tests/Test.ADAL.NET.Unit.net45/UnitTests.cs

```diff
@@ -117,6 +119,30 @@ public void AuthorityTypeDetectionTest()
             Assert.AreEqual(AuthorityType.ADFS, Authenticator.DetectAuthorityType("https://abc.com/adfs/dummy/"));
         }
 
+        [TestMethod]
+        public void AuthorityCustomPortTest()
+        {
+            using (var httpManager = new MockHttpManager())
+            {
+                var CustomPortAuthority = "https://localhost:5215/common/";
+                httpManager.AddInstanceDiscoveryMockHandler();
+                var serviceBundle = ServiceBundle.CreateWithCustomHttpManager(httpManager);
+
+                var context = new AuthenticationContext(
+                              serviceBundle,
+                              CustomPortAuthority,
+                              AuthorityValidationType.False,
+                              new TokenCache());
+
+                var result = context.AcquireTokenAsync(AdalTestConstants.DefaultRedirectUri.ToString(),
```

Comment:
We run all our tests on .NetFx and .NetCore, however you'll notice that on .net core PlatformParamters does not exist. On ADAL, you can't do acquire token interactive from .net core. So you need to not run this test on .net core.